.circleci/test_data_type.sh

```diff
@@ -2,10 +2,14 @@
 
 set -e -x
 
-cd examples/eqtl_gwas/
+cd examples/
```

Comment:
I just realized that the absolute path (`~/cimr/examples/`) will be better than relative path. When using the former, we never need to worry about from where the current script is executed. For example, when using absolute path, `test_data_type.sh` can be launched by either:
```shell
cd ~/cimr/circleci
bash ./test_data_type.sh
```
or:
```shell
bash .circleci/test_data_type.sh  # assume that we are in ~/cimr/
```
But with the relative path, this script will have to be executed ONLY when the current working directory is `~/cimr/`. Does this make sense?

Reply:
Sure, it can be changed.